### PR TITLE
Add itemadapter

### PR DESCRIPTION
**Pull request by Gallaecio**
This is a new dependency of Scrapy (https://github.com/conda-forge/scrapy-feedstock/pull/41).

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] Ping the Python maintainers, let them know this is ready for a review.

### CODE REVIEW

**Comment by Gallaecio** [comment]
@dangra @kmike @wRAR Please leave a comment confirming you are willing to be listed as a maintainer.

@synapticarbors @rmax I was not sure whether or not you would want to be added, but if you do just leave a comment as well.

Thanks!

**Comment by marcelotrevisani** [comment]
Could you please revert the files which you deleted?

**Comment by wRAR** [comment]
I'm fine with being listed as a maintainer.

File: recipes/itemadapter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/itemadapter/meta.yaml:22]:
```suggestion
    - python >=3.5
```

Just to be equal with ``host`` :)

File: recipes/itemadapter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - itemadapter
```
Comment (marcelotrevisani) [inline recipes/itemadapter/meta.yaml:26]: Can you also try to add ``pip check`` please?
```suggestion
    - itemadapter
  requires:
    - pip
  commands:
    - pip check
```

**Comment by Gallaecio** [comment]
Today I Learned about `pip check`. Thanks!

**Comment by synapticarbors** [comment]
no need to add me.

**Comment by dangra** [comment]
Fine to be listed as maintainer. thanks

**Comment by kmike** [comment]
No need to add me; I think we have enough representatives.

**Comment by Gallaecio** [comment]
That’s everyone then, @marcelotrevisani all yours!

**Pull request merged**